Ginger/Ginger/BusinessFlowWindows/xUcBusinessFlowMap.xaml.cs

```diff
@@ -0,0 +1,80 @@
+#region License
+/*
+Copyright © 2014-2019 European Support Limited
+
+Licensed under the Apache License, Version 2.0 (the "License")
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at 
+
+http://www.apache.org/licenses/LICENSE-2.0 
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS, 
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+See the License for the specific language governing permissions and 
+limitations under the License. 
+*/
+#endregion
+
+using System.Windows;
+using System.Windows.Controls;
+using GingerWPF.UserControlsLib.UCTreeView;
+using Amdocs.Ginger.Repository;
+using amdocs.ginger.GingerCoreNET;
+using GingerCore;
+using Amdocs.Ginger.Common.Enums;
+using System.Collections.Generic;
+using Ginger.SolutionWindows.TreeViewItems;
+using System;
+using Amdocs.Ginger.Common;
+
+namespace Ginger.BusinessFlowWindows
+{
+    /// <summary>
+    /// Interaction logic for xUcBusinessFlowMap.xaml
+    /// </summary>
+    public partial class xUcBusinessFlowMap : UserControl
+    {
+        public BusinessFlow mBusinessFlow;
+        public string TargetApplication { get; set; }
+
+        public xUcBusinessFlowMap()
```

Comment:
seems like there is no handling for reloading the BF object when reloading the POM after Ginger/Solution reload.
binding should not be on top of BFPath but on top of BF GUID using RepositoryITemKey class

Reply:
changed to save the RepositoryITemKey for selected businessflow